sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -2370,6 +2373,26 @@ public boolean isForceInit() {
     return forceInit;
   }
 
+  /**
+   * If set to true, automatic scope forking will be disabled. If set to false, scopes will be
+   * forked automatically, e.g. when scopes are accessed on a thread for the first time, pushScope
+   * is invoked, in some cases when we explicitly want to fork the root scopes, etc.
+   *
+   * <p>If this is set to something other than `null`, it will take precedence over what is passed
+   * to Sentry.init.
+   *
+   * <p>Enabling this is intended for mobile and desktop apps, not backends.
+   *
+   * @param globalHubMode true = automatic scope forking is disabled
+   */
+  public void setGlobalHubMode(final @Nullable Boolean globalHubMode) {
```

Comment:
Should we set it in [SentryAndroid](https://github.com/getsentry/sentry-java/blob/0cc92623d159ad340653a539366b226986e20090/sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java#L93-L158), instead of calling `Sentry.init(..., true)`?
Also, i'd add a comment in the javadoc saying that Android defaults to true

Reply:
Should do the same thing, we can replace all the init overloads with using the option in a follow up major, no rush I guess.

Will add the default docs.